app/src/main/java/org/listenbrainz/android/repository/feed/FeedRepository.kt

```diff
@@ -1,23 +1,23 @@
 package org.listenbrainz.android.repository.feed
 
-import org.listenbrainz.android.model.FeedData
-import org.listenbrainz.android.model.FeedEventDeletionData
-import org.listenbrainz.android.model.FeedEventVisibilityData
+import org.listenbrainz.android.model.feed.FeedData
+import org.listenbrainz.android.model.feed.FeedEventDeletionData
+import org.listenbrainz.android.model.feed.FeedEventVisibilityData
 import org.listenbrainz.android.model.SocialResponse
 import org.listenbrainz.android.util.Resource
 
 interface FeedRepository {
     
-    suspend fun getFeedEvents(username: String, maxTs: Int? = null, minTs: Int? = null, count: Int = 25) : Resource<FeedData>
+    suspend fun getFeedEvents(username: String?, maxTs: Int? = null, minTs: Int? = null, count: Int = 25) : Resource<FeedData>
```

Comment:
Why make username nullable at all?

Reply:
If the user isn't logged in, this field would be null

Reply:
I mean.. that was understandable already, I just wanted to say that why even check for the feed of a signed-out user?

Reply:
That is something I think we were going to figure out in future on how to ONLY allow a user when they are logged in.
On the other hand, this repository function should be foolproof in itself and not rely on forward implementations as its an isolated layer in itself.